demos/mlsys_2020/demo.ipynb

```diff
@@ -184,7 +184,10 @@
     "    ]),\n",
     "    widgets.HTML(value=\"<h2>Result</h2>\"),\n",
     "    AsyncAnimation(1, output_anim, click_pause = False),\n",
-    "])"
+    "])\n",
```

Comment:
Are you using the `pre-commit` hook that @dgkutnic added? It should normalize this stuff and remove it I think.

Reply:
Yes it should. Not sure how this got through. 
Thanks for pointing out.